ascii.js

```diff
@@ -1,35 +1,45 @@
-const clc = require('cli-color')
+var util = require('util')
```

Comment:
Variable `util` doesn't change, and can be converted to const.
```suggestion
const util = require('util')
```

Reply:
We can't. const can't be used due to v0.12 support. This is an illegal operation back in the time.

Reply:
I don't see any error with using `const` in Node.js 0.12.0, tested on Ubuntu 22.04 LTS

```console
$ node -v
v0.12.0

$ echo 'const util = require("util"); console.log(util.inspect({"foo": "bar"}));' > test.js

$ node test.js
{ foo: 'bar' }
```

It's used on the next line for variable `danger` and other variables.

Reply:
Try running the node setup-test.js.

Reply:
I don't see any errors, tested with Node.js 0.12.0 on Ubuntu 22.04 LTS

```console
$ git diff
diff --git a/ascii.js b/ascii.js
index f7ed48b..6bd80bb 100644
--- a/ascii.js
+++ b/ascii.js
@@ -1,4 +1,4 @@
-var util = require('util')
+const util = require('util')
 
 const danger = '\n' +
 '\n' +

$ node -v
v0.12.0

$ node setup-test.js
```

What errors are you seeing?

Reply:
Oh I think I got a cached error message somehow. I've pushed the changes with `const`.